web-client/iron-remote-gui/tsconfig.json

```diff
@@ -18,13 +18,7 @@
     "strictNullChecks": true,
     "noImplicitAny": true
   },
-  "include": [
-    "src/**/*.svelte",
-    "src/**/*.ts",
-    "src/**/*.js",
-    "dist/**/*.d.ts", // make sure the output file are correctly included
```

Comment:
As mentioned on slack

Reply:
Double insurance to make sure that all types that is used are defined in *.d.ts file